igamt-lite-controller/src/main/java/gov/nist/healthcare/tools/hl7/v2/igamt/lite/web/controller/SearchController.java

```diff
@@ -113,17 +97,13 @@ public Segment getSegment(@RequestParam(value="name") String name,@RequestParam(
 		return messageService.findByScope(SCOPE.HL7STANDARD.name());
 	}
 	
-	@RequestMapping(value = "/message", method = RequestMethod.GET, produces = "application/json")
-	public Message getMessage(@RequestParam(value="name") String name,@RequestParam(value="hl7Version") String hl7Version) throws DataNotFoundException {
-		if(name != null && !name.isEmpty() && hl7Version != null && !hl7Version.isEmpty()){
-			return messageService.findByNameAndVersionAndScope(name, hl7Version, SCOPE.HL7STANDARD.name());
-		}
-		return null;
-	}
-	
 	@RequestMapping(value = "/valueSets", method = RequestMethod.GET, produces = "application/json")
-	public List<Table> getValueSets(@RequestParam("scope") String scope, @RequestParam(value="bindingIdentifier", required=false) String bindingIdentifier,@RequestParam(value="hl7Version", required=false) String hl7Version) throws DataNotFoundException {
-		if(scope.equals(SCOPE.HL7STANDARD.name()) || scope.equals(SCOPE.PHINVADS.name())){
+	public List<Table> getValueSets(@RequestParam(value="scope", required=false) String scope, @RequestParam(value="bindingIdentifier", required=false) String bindingIdentifier,@RequestParam(value="hl7Version", required=false) String hl7Version) throws DataNotFoundException {
+	  //By default, the scope is HL7
+	  if(scope == null){
```

Comment:
No the default is chosen on the client side.  The server needs a value. If no value is provided the server must return nothing.